postgresql-11.5/src/bin/psql/commandbteq.c

```diff
@@ -345,43 +347,29 @@ static dotResult
 exec_command_set(BteqScanState scan_state, bool active_branch)
 {
     bool        success = true;
-
+    char        quote;
     if (active_branch)
     {
-        char       *opt0 = bteq_scan_dot_option(scan_state,
-                                                  OT_BTEQ_NORMAL, NULL, false);
-
-        if (!opt0)
+        char *opt0 = read_connect_arg(scan_state);
+        if (opt0)
```

Comment:
Let's define variables with meaningful names and do not use digits in the variable names.
This could be defined as `width_keyword`, the `opt` could be `width_value`.
It makes your code more readable and easy to review.